### PR TITLE
UIBULKED-292: Arrow icons in "ID", "# of records" and "Processed" columns remain visible after resetting filters

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
Arrow icons in "ID", "# of records" and "Processed" columns remain visible after resetting filters

## Purpose

[UIBULKED-292](https://issues.folio.org/browse/UIBULKED-292)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by zburke — CHANGES_REQUESTED** [review]
Please add a `useLocationSorting.test.js` file to add coverage for this new function. 

Is it possible to improve those imports from stripes-acq-components as well? It would be really nice to avoid importing by private path.

File: src/hooks/useLocationSorting.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import useSorting from '@folio/stripes-acq-components/lib/AcqList/hooks/useSorting';
+import { useCallback, useEffect } from 'react';
+import { buildSearch, buildSortingObj } from '@folio/stripes-acq-components/lib/AcqList/utils';
```
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/hooks/useLocationSorting.js:3]: everything that is placed in AcqList can be exported from `'@folio/stripes-acq-components'`, no idea why private path are used here

File: src/hooks/useLocationSorting.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+export const useLocationSorting = (location, history, resetData, sortableFields, defaultSorting) => {
```
Comment (NikitaSedyx) [inline src/hooks/useLocationSorting.js:5]: what the difference between this hooks and hooks from acq-components?

Comment (UladzislauKutarkin) [reply src/hooks/useLocationSorting.js:5]: @NikitaSedyx it has deps in useEffect, in acq-components it has empty deps

File: src/hooks/useLocationSorting.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  ] = useSorting(resetData, sortableFields);
+
+  useEffect(
+    () => {
+      const initialSorting = buildSortingObj(location.search, defaultSorting);
+
+      setSortingField(initialSorting.sortingField);
+      setSortingDirection(initialSorting.sortingDirection);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
[the remark this answers is not in the document]
Comment (UladzislauKutarkin) [reply src/hooks/useLocationSorting.js:21]: Thanks for that, I'll remove that

File: src/hooks/useLocationSorting.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (UladzislauKutarkin) [reply src/hooks/useLocationSorting.js]: Fixed, only `useSorting` we can't import from `@folio/stripes-acq-components`

File: src/hooks/useLocationSorting.js (hunk 2 above)
Comment (NikitaSedyx) [reply src/hooks/useLocationSorting.js:11]: does it make sense to fix in acq-components?

File: src/hooks/useLocationSorting.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (NikitaSedyx) [reply src/hooks/useLocationSorting.js]: does it make sense to export it there?

File: src/hooks/useLocationSorting.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/hooks/useLocationSorting.js:11]: Looks like we can pass deps in `acq-components` and pass it, and we don't need it make duplicate of that hook in `bulk-edit`

File: src/components/BulkEditLogs/BulkEditLogs.js [hunk 4]
```diff
@@ -10,13 +10,17 @@ import {
...
   PrevNextPagination,
-  useLocationSorting,
   usePagination,
 } from '@folio/stripes-acq-components';
 
+import {
+  DESC_DIRECTION,
+  SORTING_DIRECTION_PARAMETER,
+  SORTING_PARAMETER,
+} from '@folio/stripes-acq-components/lib/AcqList/constants';
```
Comment (zburke) [inline src/components/BulkEditLogs/BulkEditLogs.js:20]: Can we fix the private-path imports for these constants, too?

File: src/hooks/useLocationSorting.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/hooks/useLocationSorting.js]: @zburke , @NikitaSedyx  I made changes in acq components https://github.com/folio-org/stripes-acq-components/pull/695 so we don't need our custom `useLocationSorting.js`

**Review by vashjs — APPROVED** [review]

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**